aws/agent_fargate.yaml

```diff
@@ -40,7 +40,7 @@ Parameters:
     Default: ""
   Image:
     Description: The OtterTune agent image
-    Default: public.ecr.aws/ottertune/agent:0.1.6
+    Default: public.ecr.aws/ottertune/agent:0.2.1
```

Comment:
Do we need the default value here? I think it's passed by the frontend ?

Reply:
Yeah may just be able to get rid of it, Nate asked me to update this previously based on in case someone creates a stack from CF directly instead of via our site.